packages/fluentui/react-northstar/src/themes/teams-dark-v2/components/Chat/chatMessageVariables.ts

```diff
@@ -0,0 +1,15 @@
+import { ChatMessageVariables } from '../../../teams/components/Chat/chatMessageVariables';
+
+export const chatMessageVariables = (siteVars: any): Partial<ChatMessageVariables> => {
```

Comment:
nit: do we not have a type for siteVars ?

Reply:
I haven't found any, also we are importing them just by

```
import * as siteVariables from './siteVariables';
```

Reply:
Alright, might make sense to start typing this in the future even just for intellisense/autocomplete.

Doesn't really impact your PR so feel free to resolve